content/blog/pulumi-now-supports-atlassian-identity/index.md

```diff
@@ -1,21 +1,22 @@
 ---
 title: "Pulumi now supports Atlassian Identity"
 authors: ["praneet-loke"]
-tags: [""]
+tags: ["New-Feature"]
```

Comment:
A couple posts are tagged "New-Feature" while others are tagged "New-Features" (plural).

<img width="325" alt="Screen Shot 2019-06-19 at 3 10 50 PM" src="https://user-images.githubusercontent.com/710598/59805061-11f46800-92a5-11e9-8834-6f1a00b10076.png">

Do we want to standardize on one of them?

Reply:
Good catch, I'll fix this `New-Feature` seems legit.